vendor/qtidataservices_app.te

```diff
@@ -0,0 +1,20 @@
+type qtidataservices_app, domain;
+app_domain(qtidataservices_app)
+net_domain(qtidataservices_app)
+
+get_prop(qtidataservices_app, vendor_default_prop)
+set_prop(qtidataservices_app, exported_radio_prop)
+
+# Find the vendor.qti.hardware.data.iwlan::IIWlan HIDL service
+# And grant binder access to the host (`rild`, `cnd`)
+hal_client_domain(qtidataservices_app, hal_telephony)
+allow qtidataservices_app radio_service:service_manager find;
```

Comment:
Yes, it is indeed trying to access `isub`.

Reply:
Feel free to copy the documentation-comments from aforementioned domains to indicate this.

However, it seems these have been removed in the force-push, is it not accessed after all?

Reply:
It appeared before but now it's gone /shrug, guess qcrilam and other apps do all the hard work making this policy excessive.

Reply:
It might be one-time setup; sepolicy is tricky to manage around those (and it's even harder to accurately determine whether an application should logically receive certain exemptions). I generally wipe my device and boot with enforcing from the get-go after initial policy bringup to validate soundness. Not sure my test results would be valid here without a carrier supporting IMS and all the various tidbits around it.